crates/circuit/src/bit.rs

```diff
@@ -771,6 +771,25 @@ macro_rules! create_bit_object {
                 })
             }
 
+            /// Allows for the creation of a new register with a provisional prefix and the
```

Comment:
Nitpicking, but the word "provisional" here feels a bit unclear to me, since it makes me think the name is temporary.